pkg/agent/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -54,6 +54,9 @@ const (
 //              b. Notify dirty rules
 //
 type Controller struct {
+	// antreaPolicyEnabled indicates whether Antrea NetworkPolicy and
+	// ClusterNetworkPolicy are enabled.
+	antreaPolicyEnabled bool
```

Comment:
btw why not use features.DefaultFeatureGate.Enabled() to determine?

Reply:
bool is more efficient than checking a string? I am just following other code.

Reply:
I guess the point is we can import "github.com/vmware-tanzu/antrea/pkg/features" and check for whether this feature is enabled w/o having to have agent.go pass in the value in NewNetworkPolicyController?

Reply:
Understood, but I think that means to check the "AntreaPolicy" string every time?